public/js/components/Editor.css

```diff
@@ -90,4 +90,9 @@
   background-color: var(--theme-tab-toolbar-background);
   font-weight: lighter;
   z-index: 100;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -o-user-select: none;
+  user-select: none;
```

Comment:
Does this apply it to the text within CodeMirror? The code should be selectable.